e2e/tests/ui/features/@search/search.step.ts

```diff
@@ -0,0 +1,275 @@
+import { SearchPage } from "../../helpers/SearchPage";
+import { ToolbarTable } from "../../helpers/ToolbarTable";
+import { Tabs } from "../../helpers/Tabs";
+import { DetailsPage } from "../../helpers/DetailsPage";
+import { createBdd } from "playwright-bdd";
+import { expect } from "@playwright/test";
+
+export const { Given, When, Then } = createBdd();
+
+/**
+ * This function returns table identifier and column, which contains link to the details page
+ * @param type Catogory of the data to get the table identifier and column
+ */
+function getTableInfo(type: string): [string, string] {
```

Comment:
Helper functions should not be defined in the step definition file. Please use the methods defined for the page objects as mentioned above, if applicable, and if you have to extend them, please do it there.